runtime/error.c

```diff
@@ -1,12 +1,103 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
 #include "balrt.h"
+#include "third-party/libbacktrace/backtrace.h"
+
+#define INITIAL_PC_COUNT 1
+#define MAX_PC_COUNT 256
+#define SKIP_FROM_END 1 // This number counts from the last stack frame
```

Comment:
This should be a parameter at the time you print out

Reply:
This is a parameter that we need to pass `backtrace_simple` and without this, libbacktrace generates an entry for `backtrace_simple` function too. So shall we keep it like this?